docs_src/design/adr/template.md

```diff
@@ -0,0 +1,70 @@
+# Architecture Decision Record (ADR) template <!-- Replace with ADR title -->
+This is a template for EdgeX ADR.
+The description of each section is included as comments and visible only when viewing the [source](https://raw.githubusercontent.com/edgexfoundry/edgex-docs/main/docs_src/design/adr/template.md).
+Remove this paragraph before submitting the ADR.
+
+### Submitters
+<!-- List ADR submitters
+
+Format:
+- Name (Organization)
+-->
+
+## Change Log
+<!-- List the changes to the document, incl. state, date, and PR URL.
+State is one of: accepted, amended, deprecated

```

Comment:
```suggestion
State is one of:  pending, accepted, amended, deprecated

```

Reply:
Is this meant for the authoring and PR review phase?

Reply:
I find it hard to understand the use of a pending/proposed/under-review state. The state is part of a change log which is only effective when PR has merged. Maybe I'm wrong, but I don't think a document with such state ever make it into the records. The repository maintainer (TCS chair?) should add the state before merging (if allowed by the PR submitter) or afterwards and then add the record to ToC.

If we do add that state, what will the date be? Maybe the state could be "submitted" and have a date. Still, the PR URL will need to be added after PR has been created.

Another option would be to add a note to say this section should be left alone during PR submission and review.

Reply:
Pending is probably not necessary in that the status is just not accepted when the PR first gets created.  But sometimes people feel the need to fill in a field with something. Pending works for this.